test/functional/cypress/specs/companies/referrals/send-referral-spec.js

```diff
@@ -42,50 +42,51 @@ describe('Send a referral form', () => {
       })
 
       it('should display the headings and four fields', () => {
-        cy.get(selectors.companySendReferral.form)
-          .find('h4')
-          .should('contain', 'Who do you want to refer this company to?')
-          .next()
+        cy.contains('Who do you want to refer this company to?').should(
+          'be.visible'
+        )
+        cy.get('label')
+          .eq(0)
           .should('contain', 'Adviser')
```

Comment:
Could be simplified with:
```suggestion
        cy.contains('label', 'Adviser')
```

There is a few more tests that can be simplified with this method.